docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -0,0 +1,189 @@
+
+<details><summary style="font-size:20px;font-weight:bold">Get invited to TechPass account and request for SEED (optional)</summary>
+
+1. Contact your project manager or the reporting officer to request for TechPass account and SEED provisioning(optional).
+
+!> You need SEED provisioning to access SGTS and GCC 2.0 resources via an Internet Device.
+
+2. Provide the required details in this request such as your organisational email address, mobile phone number and project name.
+
+   Project manager or the reporting officer contacts the sponsoring agency or the tenant admin to invite you to TechPass.
+
+<!--3. If you are requesting for SEED provisioning, provide the details of your Internet Device.
+
+Your project manager or reporting officer will work with the sponsoring government agency to provide the requested services.
+-->
+
+> **Additional information**:
+  >
+  > **When you are invited to TechPass**
+  >- A TechPass account is provisioned for you and is in pending state.
+  >- We'll send the TechPass onboarding email to activate the account. This email is valid only for 30 days and will have your TechPass account or log in ID. 
+  >- Your TechPass log in ID's domain will be ``techpass.gov.sg```.
+  
+  >
+  > **If SEED provisioning is approved**:
+  >- We'll send the SEED onboarding email within the next three business days.
+  >- This email is valid only for 30 days.
+  >- Ensure that you have activated your TechPass account before proceeding to onboard your Internet Device to SEED .
+  >- If your SEED onboarding email has expired, you need to request again.
+
+ 
+</details>
+
+## Step 2. Sign in using initial password
+
+<details> <summary style="font-size:20px;font-weight:bold">Sign in to TechPass using your TechPass username and initial password</summary>
+
+ 1. Go to the required Docs portal environment and click **Login**.
+
+    - [Docs portal - staging environment](https://stg.docs.developer.tech.gov.sg/)
+    - [Docs portal - production environment](https://docs.developer.tech.gov.sg/)
+
+  2. Enter your TechPass username and click **Next**.
+
+    ![vendor-sign-in-1](assets/support/Vendor_email.png)
+
+  3. Enter the initial password and click **Sign in**.
+
+    ![vendor-initial-pwd](assets/support/vendor-initial-password.png)
+
+  4. Click **Next** to configure MFA for your TechPass account. 
+
+   ![proceed-to-mfa-setup](assets/support/more-info-required.png ':size=500')
+
+  </details>
+
+## Step 3. Configure and verify MFA for TechPass account
+
+<details> <summary style="font-size:20px;font-weight:bold">Set up security verification for TechPass account</summary>
+
+  1. Install Microsoft Authenticator on your mobile device.
+  
+  2. Click **Next** on your computer. 
+
+    ![vendor-mfa-1](assets/support/vendor-mfa-1-new.png)
+
+  > **Note**
+  > You may install any authenticator. However, as we recommend Microsoft authenticator, this document guides you to configure TechPass MFA using that.
```

Comment:
Since we are recommending the users to use Microsoft Authenticator let's indicate the reasons, for example:
1. It supports Number Matching to protect the user from MFA Fatigue attacks.
2. Microsoft constantly improve their MFA security to protect the user and it may only apply to users using Microsoft Authenticator.

Reply:
done!